### PR TITLE
PILOT-6142: Add FileOps service helm charts

### PR DESCRIPTION
**Pull request by juyoungkimthedev**
This PR adds new FileOps service related helm charts and configurations that is integration of download and upload services into one single service, thus some ENVs and configurations are combined too.

### CODE REVIEW

File: fileops-service/values.yaml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+appConfig:
+  port: 5078
+  env: "dev"
+  config_center_enabled: false
+  config_center_base_url: http://common.utility:5062/
+  namespace: ""
+
+  ROOT_PATH: "/data/core-storage"
+
+  LEVEL_DEFAULT: 30 # WARN
```
Comment (acascais) [inline fileops-service/values.yaml:22]: as defined in line 67-74 of deployment.yaml, these vars should be "LOG_LEVEL_X"

File: fileops-service/values.yaml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+autoscaling:
+  enabled: false
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
+
+readinessProbe: {}
```
Comment (acascais) [inline fileops-service/values.yaml:80]: we should defined default readiness and liveness probes

File: fileops-service/values.yaml [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+securityContext: {}
+
+service:
+  type: ClusterIP
+  port: 80
+
+ingress:
+  enabled: false
+
+resources: {}
```
Comment (acascais) [inline fileops-service/values.yaml:69]: highly recommend setting resources in the defaults.

if you don't know/can't predict how much cpu&memory resources this service will have, i recommend setting higher values for now and then we will adjust later in the helmfile values.yaml:
```
resources:
  limits:
    cpu: 300m
    memory: 300Mi
  requests:
    cpu: 100m
    memory: 100Mi
    ```

File: fileops-service/templates/service.yaml [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+kind: Service
+metadata:
+  name: {{ include "fileops-service.fullname" . }}
+  labels:
+    {{- include "fileops-service.labels" . | nindent 4 }}
+spec:
+  type: {{ .Values.service.type }}
+  ports:
+    - port: {{ .Values.service.port }}
+      targetPort: {{ .Values.service.targetPort }}
```
Comment (acascais) [inline fileops-service/templates/service.yaml:11]: this is not defined in the values.yaml file. It definitely should.

Comment (juyoungkimthedev) [reply fileops-service/templates/service.yaml:11]: I have defined as

```
service:
  type: ClusterIP
  port: 5078
  targetPort: 5078
```

File: fileops-service/values.yaml (hunk 3 above)
Comment (acascais) [reply fileops-service/values.yaml]: Different services need different resources. We don't know what resources this service will need yet, but that's okay: now we can configure them in the values file in the helmfile repo.

File: fileops-service/values.yaml [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+
+tolerations: []
+
+affinity: {}
+
+readinessProbe:
+  httpGet:
+   path: /v1/health
+   port: 5078
+  tcpSocket:
```
Comment (andricDu) [inline fileops-service/values.yaml:91]: We will want to remove the tcp socket probe I believe.

**Review by acascais — APPROVED** [review]

**Pull request merged**